src/commands/database/init.ts

```diff
@@ -0,0 +1,181 @@
+import { OptionValues } from 'commander'
+import inquirer from 'inquirer'
+import BaseCommand from '../base-command.js'
+import { getAccount, getExtension, getJigsawToken, getPackageJSON, installExtension, spawnAsync } from './utils.js'
+import { initDrizzle } from './drizzle.js'
+import { NEON_DATABASE_EXTENSION_SLUG, NETLIFY_NEON_PACKAGE_NAME } from './constants.js'
+import prettyjson from 'prettyjson'
+import { log } from '../../utils/command-helpers.js'
+import { SiteInfo } from './database.js'
+
+export const init = async (_options: OptionValues, command: BaseCommand) => {
+  const siteInfo = command.netlify.siteInfo as SiteInfo
+  if (!command.siteId) {
+    console.error(`The project must be linked with netlify link before initializing a database.`)
+    return
+  }
+
+  const initialOpts = command.opts()
+
+  const opts = command.opts<{
+    drizzle?: boolean | undefined
+    overwrite?: boolean | undefined
+    minimal?: boolean | undefined
+  }>()
+
+  if (!command.netlify.api.accessToken || !siteInfo.account_id || !siteInfo.name) {
+    throw new Error(`Please login with netlify login before running this command`)
+  }
+
+  if (opts.minimal === true) {
+    command.setOptionValue('drizzle', false)
+  }
+
+  const account = await getAccount(command, { accountId: siteInfo.account_id })
+
+  const netlifyToken = command.netlify.api.accessToken.replace('Bearer ', '')
+
+  const extension = await getExtension({
+    accountId: siteInfo.account_id,
+    netlifyToken: netlifyToken,
+    slug: NEON_DATABASE_EXTENSION_SLUG,
+  })
+  if (!extension?.hostSiteUrl) {
+    throw new Error(`Failed to get extension host site url when installing extension`)
+  }
+
+  const installNeonExtension = async () => {
+    if (!account.name) {
+      throw new Error(`Failed to install extension "${extension.name}"`)
+    }
+    const installed = await installExtension({
+      accountId: siteInfo.account_id,
+      netlifyToken: netlifyToken,
+      slug: NEON_DATABASE_EXTENSION_SLUG,
+      hostSiteUrl: extension.hostSiteUrl,
+    })
+    if (!installed) {
+      throw new Error(`Failed to install extension on team "${account.name}": "${extension.name}"`)
+    }
+    log(`Extension "${extension.name}" successfully installed on team "${account.name}"`)
```

Comment:
Now now, but in the future: Should we replace this with a call to the extension host site and let the host install the extension, so we're not re-implementing this logic in quite so many places?